src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -604,15 +628,15 @@ void EnDns_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Ve
 }
 
 void EnDns_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static TexturePtr D_8092DE1C[] = { object_dns_Tex_0028E8, object_dns_Tex_002968, object_dns_Tex_0029E8,
-                                       object_dns_Tex_002968 };
+    static TexturePtr sEyeTextures[] = { gKingsChamberDekuGuardEyeOpenTex, gKingsChamberDekuGuardEyeHalfTex,
+                                         gKingsChamberDekuGuardEyeClosedTex, gKingsChamberDekuGuardEyeHalfTex };
```

Comment:
```suggestion
    static TexturePtr sEyeTextures[] = { gKingsChamberDekuGuardEyeOpenTex, gKingsChamberDekuGuardEyeHalfTex,
                                         gKingsChamberDekuGuardEyeClosedTex, gKingsChamberDekuGuardEyeHalfTex, };
```
With the textures named, it would format better with a trailing comma.